server/chat-plugins/datasearch.js

```diff
@@ -1212,12 +1255,34 @@ function runMovesearch(target, cmd, canAll, message) {
 	}
 	if (results.length > 1) {
 		results.sort();
+		if (sort) {
+			let prop = sort.slice(0, -1);
+			let direction = sort.slice(-1);
+			results.sort((a, b) => {
+				let move1 = dex[toId(direction === '+' ? a : b)], move2 = dex[toId(direction === '+' ? b : a)];
+				if (move1[prop] === true) {
```

Comment:
Can you just convert `true` to `1` and `false` to `0`? Then you can return `move1prop - move2prop` without any special code.